package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Alexander Feiglstorfer <af@storyblok.com>",
   "dependencies": {
     "json-stringify-safe": "^5.0.1",
-    "storyblok-js-client": "^2.5.2"
+    "storyblok-js-client": "^3.3.1"
```

Comment:
Please, add `axios` as dependency.

Reply:
Here you go: https://github.com/storyblok/gatsby-source-storyblok/pull/41

Reply:
@emanuelgsouza I've added the Axios dependency

Reply:
Thanks @terhuerne. I'll close my PR.